Firebase/Auth/Source/AuthProvider/OAuth/FIROAuthProvider.m

```diff
@@ -165,6 +166,9 @@ - (void)getCredentialWithUIDelegate:(nullable id<FIRAuthUIDelegate>)UIDelegate
     @return An Instance of @c FIROAuthProvider.
   */
 - (nullable instancetype)initWithProviderID:(NSString *)providerID auth:(FIRAuth *)auth {
+  NSCAssert(![providerID isEqual:FIRFacebookAuthProviderID],
```

Comment:
This should be NSAssert instead of NSAssert. Is the consideration here that being broken for debug builds is okay but we don't want to crash release builds?

Reply:
Yeah.